src/main/java/com/exasol/releasedroid/adapter/github/RepositoryStructureValidator.java

```diff
@@ -9,75 +12,78 @@
 import java.util.logging.Logger;
 
 import com.exasol.errorreporting.ExaError;
-import com.exasol.releasedroid.adapter.AbstractRepositoryValidator;
 import com.exasol.releasedroid.usecases.report.Report;
 import com.exasol.releasedroid.usecases.report.ValidationResult;
 import com.exasol.releasedroid.usecases.repository.ReleaseLetter;
 import com.exasol.releasedroid.usecases.repository.Repository;
+import com.exasol.releasedroid.usecases.validate.StructureValidator;
 
 /**
- * Contains validations for a Git project.
+ * Contains validations for a repository structure.
  */
-public class GitHubRepositoryValidator extends AbstractRepositoryValidator {
-    private static final Logger LOGGER = Logger.getLogger(GitHubRepositoryValidator.class.getName());
-    protected static final String PREPARE_ORIGINAL_CHECKSUM_WORKFLOW_PATH = ".github/workflows/release_droid_prepare_original_checksum.yml";
-    protected static final String PRINT_QUICK_CHECKSUM_WORKFLOW_PATH = ".github/workflows/release_droid_print_quick_checksum.yml";
+public class RepositoryStructureValidator implements StructureValidator {
```

Comment:
Maybe we can call this by it's distinctive factor, as these validations have to pass for every repository on top of the specific ones  (Java, Scala, Python).
Maybe something like `CommonRepositoryValidator` or `ReleasableRepositoryValidator` or something similar?